policy/policy.go

```diff
@@ -78,6 +88,9 @@ func (p *Policy) GetReserveOnchainMsat() uint64 {
 // IsPeerAllowed returns if a peer or node is part of
 // the allowlist.
 func (p *Policy) IsPeerAllowed(peer string) bool {
+	if p.IsPeerSuspicious(peer) {
+		return false
```

Comment:
This is simple but this would be indistinguishable from an unallowed request rejection in the logs? Ideally we want:

- [ ] The rejection reason to be logged so the admin knows they need to inspect what caused the peer to become sus.
- [ ] Unallowed vs sus be communicated within the error message so the requesting peer knows why.

It's OK to commit as-is if this would be a lot of extra work. We'll fine it as another ticket.

Reply:
Check new push